src/auto-complete.jsx

```diff
@@ -152,6 +157,7 @@ const AutoComplete = React.createClass({
       open: false,
       searchText: '',
       menuCloseDelay: 100,
+      menuItemsLimit: 20,
```

Comment:
I'm having second though. What using `-1` as the default value?
This way, we won't introduce any breaking changes.